### PR TITLE
Bash completions.

### PR DESCRIPTION
**Pull request by mchav**
Bash completions for all commands in usage. Subcommands not yet supported.

### CODE REVIEW

File: completions/cbt [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    opts="_context apiTarget assertSourceDirectories c classLoaderCache \
+          classpath compile compileStatusFile compileTarget context crossScalaVersions \
+          crossScalaVersionsArray defaultScalaVersion defaultVersion dependencies dependenciesArray \
+          dependencyClasspath dependencyClasspathArray dependencyTree description developers docJar \
+          enableConcurrency exportedClasspath exportedClasspathArray finalBuild jar jarTarget lib \
+          licenses localJars logger name needsUpdate needsUpdateCompat package pom pomExtra projectDirectory \
+          publishSigned publishSnapshot r recursive releaseFolder releaseUrl rt run runClass scalaMajorVersion \
+          scalaTarget scalaVersion scalacOptions scmConnection scmUrl show snapshotUrl sonatypeCredentials \
+          sourceFiles sources srcJar t target targetClasspath test tools transitiveDependencies \
+          triggerLoopFiles triggerLoopFilesArray url usage version zincVersion"
```
Comment (cvogt) [inline completions/cbt]: there should be a way to fetch these dynamically from `cbt`

File: completions/cbt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    COMPREPLY=( $(compgen -W "${opts}" -- ${cur}) )
+    return 0
+}
+
+complete -F _cbt cbt
```
Comment (cvogt) [inline completions/cbt]: So if we adapt this like this: http://unix.stackexchange.com/a/227862/114989 we could give dynamic completions

**Comment by mchav** [comment]
@cvogt dynamic completions look fine now.

**Comment by cvogt** [comment]
closed in favor of #211

**Pull request closed without merging**